recipe/meta.yaml

```diff
@@ -1,55 +1,59 @@
 {% set name = "flower" %}
-{% set version = "0.9.1" %}
-{% set md5 = "5d191dae0db989b97d9d000133ead843" %}
-{% set build = 0 %}
+{% set version = "0.9.2" %}
+{% set sha256 = "a7a828c2dbea7e9cff1c86d63626f0eeb047b1b1e9a0ee5daad30771fb51e6d0" %}
 
 package:
-    name: {{ name }}
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    md5: {{ md5 }}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-    number: {{ build }}
+    number: 0
     script: python setup.py install --single-version-externally-managed --record=record.txt
     entry_points:
         - flower = flower.__main__:main
 
 requirements:
-    build:
-        - python
-        - setuptools
-        - celery >=2.5.0
-        - tornado >=4.0.0
-        - babel
-        - pytz
-        - futures
-    run:
-        - python
-        - setuptools
-        - celery >=2.5.0
-        - tornado >=4.0.0
-        - babel
-        - pytz
-        - futures
+  build:
+    - python
+    - setuptools
+    - celery >=3.1.0
+    - tornado >=4.2.0
+    - babel >=1.0
+    - pytz
```

Comment:
Expect we can can drop everything from here except `python` and `setuptools`.

Reply:
I agree. I didn't want to change it too much from the original. I'll do it.

Reply:
Thanks.

note: GitHub moved the comment from the `build` section annoyingly. IOW this is resolved and can be ignored.